subsys/spm/spm.c

```diff
@@ -59,23 +59,11 @@
  *        |      Flash          |
  *  0 kB  |---------------------|
  *
- *  * The security configuration for SRAM is applied:
+ *  * The SRAM configuration is given by the partition manager.
+ *  * To see the current configuration, run the 'pm_report' target.
```

Comment:
pm_report -> rom_report?

Reply:
I will create a separate PR adding a pm_report, this because I don't think we should print the RAM configuration when performing the rom_report.